code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -0,0 +1,239 @@
+
+#define BEEBOX_MAX_FRAMES				3		//Max frames per box
+#define BEES_RATIO						0.5 	//Multiplied by the max number of honeycombs to find the max number of bees
+#define BEE_PROB_NEW_BEE				20		//The chance for spare bee_resources to be turned into new bees
+#define BEE_RESOURCE_HONEYCOMB_COST		100		//The amount of bee_resources for a new honeycomb to be produced, percentage cost 1-100
+#define BEE_RESOURCE_NEW_BEE_COST		50		//The amount of bee_resources for a new bee to be produced, percentage cost 1-100
+
+
+
+/mob/proc/bee_friendly()
+	return 0
+
+
+/mob/living/simple_animal/hostile/poison/bees/bee_friendly()
+	return 1
+
+
+/mob/living/carbon/human/bee_friendly()
+	if(dna && dna.species && dna.species.id == "pod") //bees pollinate plants, duh.
+		return 1
```

Comment:
@Incoming5643 I look into your way first, there are 4 lines that reference ```ignored_by```
1. the definition of the list
2. it being set for slime people to be immune to slimes
3. it being set for shadowlings to be immune to faithless
4. it being used so slimes don't attack slime people

it literally isn't used outside of slimes, and I wasn't in the mood to shoehorn it into hostile AI.